website/docs/expressions/_user.md

```diff
@@ -0,0 +1,10 @@
+- `user`: The current user. This may be `None` if there is no contextual user. See ([User](../user-group/user.md#object-attributes))
+
+Example:
+
+```python
+if not 'custom_attribute' in request.user.attributes:
+  return {'custom_attribute': 'default'}
+
+return {"custom_attribute": request.user.attributes['custom_attribute']}
```

Comment:
it would probably be cleaner to use something like 

```python
return {
  "custom_attribute": request.user.attributes.get("custom_attribute", "default"),
}
```

I'd also add `user.group_attributes()`, which returns a dict of all the user's group's attributes, merged